setup.py

```diff
@@ -76,6 +82,7 @@ def _get_dbt_core_version():
     install_requires=[
         "dbt-core~={}".format(dbt_core_version),
         "sqlparams>=3.0.0",
+        "pyhive[hive_pure_sasl]~=0.7.0"
```

Comment:
If we need this for install in general, do we need it in `pyhive_extras`?

Reply:
my bad, removed from `install_requires`